client/src/RatingsAndReviews/RatingsBreakdown/BreakdownClickable.jsx

```diff
@@ -1,13 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { reviewsInteraction } from '../../clientRoutes/reviews.js';
 
 const BreakdownClickable = (props) => {
   return (
     <div className="breakdown-clickable">
       <div className="bar-label">{props.stars} stars</div>
-      <div className="bar-wrapper" star={props.stars} onClick={props.starFilterClick}>
-        <div className="bar-green" star={props.stars} style={{width: `${props.percent}%`}}></div>
-        <div className="bar-grey" star={props.stars} style={{width: `${100 - props.percent}%`}}></div>
+      <div className="bar-wrapper"
+        interaction={`${props.stars} star bar clicked`}
```

Comment:
already was mentioned to you - but it seems like you have the functionality for the change request for adding click tracker - refactoring out to renderProps will complete this for you